service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ImmunizationObservationStatementMapperTest.java

```diff
@@ -232,12 +247,44 @@ private static Stream<Arguments> resourceFileParams() {
     }
 
     @Test
-    public void When_MappingImmunizationWithInvalidPractitionerReferenceType_Expect_Error() throws IOException {
+    public void When_MappingImmunizationWithInvalidPractitionerReferenceType_Expect_Error() {
```

Comment:
Typically with using JUnit, tests should be package-private - could we omit the `public` access modifier please, and repeat throughout this test document?

Reply:
@martin-nhs 
<img width="842" alt="image" src="https://github.com/user-attachments/assets/7cdbaa00-6fe5-4245-a6eb-4667c496f152">